src/main/scala/cromwell/engine/backend/jes/GcsAuth.scala

```diff
@@ -0,0 +1,71 @@
+package cromwell.engine.backend.jes
+
+import play.api.libs.json._
```

Comment:
I thought the Spray JSON was used only in the WS API and I didn't even take a good look at it actually but you're right it's stupid to add another library if we already have one. I'll change this to use Spray it shouldn't be too bad.